libraries/src/Image/Image.php

```diff
@@ -248,109 +248,110 @@ private static function getOrientationString(int $width, int $height): string
 	}
 
 	/**
-	 * Method to generate thumbnails from the current image. It allows creation by resizing or cropping the original image.
+	 * Method to generate different sized versions of current image.
```

Comment:
Still: "It allows creation by resizing or cropping the original image."
Missing in descr.